core/arch/arm/plat-imx/config/config_imx6multi.h

```diff
@@ -139,6 +119,10 @@
 
 /* define the memory areas */
 
+#define TZDRAM_BASE			(CFG_DDR_TEETZ_RESERVED_START)
```

Comment:
This introduce `CFG_DDR_TEETZ_RESERVED_START`/`_SIZE` as config option which were not previously. I will discard such change. Maintainers will be free to later add support for external config means (`CFG_DDR_TEETZ_RESERVED_START` or `CFG_TZDRAM_BASE`) if they will.